ievaaleksandravica/nodejs-course
# installing sendgrid and sending test email

* Exploring Sendgrid
   * sendgrid.com offers free version and does not require a custom domain.
   * under Email API -> Integration Guid -> Web API -> Node.js
      * provide API key (random name), then this will create a key, add it to your confid file.
   * new directory `emails` and file `account.js`
      * import the API key from the config file
   * npm module
      * install `npm i @sendgrid/mail@7.6.0`
      * require `const sgMail = require('@sendgrid/mail')`
      * pass API key `sgMail.setApiKey(sengridApiKey)`
   * verification
      * go to settings -> sender authentication -> single sender verification
      * complete the setup and confirm the email
   * send email
      * use `sgMail.send({})` method
      * pass `to:, from:, subject:, text:` information
   * verify 
      * whether in SendGrid interface, following the verification step or checking your inbox
      * might end up in spam, because it is supposably being sent from your account
